bug: verifier should not thow IndexError

Example stacktrace:

```sehll
.0 = <enumerate object at 0x1151ada00>

>   if all(r == calls_subset[j] for j, r in enumerate(rehearsals)):
E   IndexError: list index out of range

../../../.local/share/virtualenvs/api-7-r63_lO/lib/python3.7/site-packages/decoy/verifier.py:24: IndexError
```

--- a/decoy/verifier.py
+++ b/decoy/verifier.py
@@ -21,7 +21,8 @@ class Verifier:
         else:
             for i, call in enumerate(calls):
                 calls_subset = calls[i : i + len(rehearsals)]
-                if all(r == calls_subset[j] for j, r in enumerate(rehearsals)):
+
+                if calls_subset == rehearsals:
                     return None
 
         raise VerifyError(

--- a/tests/test_verifier.py
+++ b/tests/test_verifier.py
@@ -46,6 +46,17 @@ verify_raise_specs = [
     VerifySpec(
         rehearsals=[
             VerifyRehearsal(spy_id=101, spy_name="spy_101", args=(1, 2, 3), kwargs={}),
+            VerifyRehearsal(spy_id=101, spy_name="spy_101", args=(4, 5, 6), kwargs={}),
+            VerifyRehearsal(spy_id=202, spy_name="spy_202", args=(7, 8, 9), kwargs={}),
+        ],
+        calls=[
+            SpyCall(spy_id=101, spy_name="spy_101", args=(1, 2, 3), kwargs={}),
+            SpyCall(spy_id=101, spy_name="spy_101", args=(4, 5, 6), kwargs={}),
+        ],
+    ),
+    VerifySpec(
+        rehearsals=[
+            VerifyRehearsal(spy_id=101, spy_name="spy_101", args=(1, 2, 3), kwargs={}),
         ],
         calls=[
             SpyCall(spy_id=101, spy_name="spy_101", args=(1, 2, 3), kwargs={}),
@@ -71,6 +82,19 @@ verify_pass_specs = [
             VerifyRehearsal(spy_id=202, spy_name="spy_202", args=(7, 8, 9), kwargs={}),
         ],
         calls=[
+            SpyCall(spy_id=101, spy_name="spy_101", args=(1, 2, 3), kwargs={}),
+            SpyCall(spy_id=101, spy_name="spy_101", args=(4, 5, 6), kwargs={}),
+            SpyCall(spy_id=202, spy_name="spy_202", args=(7, 8, 9), kwargs={}),
+        ],
+    ),
+    VerifySpec(
+        rehearsals=[
+            VerifyRehearsal(spy_id=101, spy_name="spy_101", args=(1, 2, 3), kwargs={}),
+            VerifyRehearsal(spy_id=101, spy_name="spy_101", args=(4, 5, 6), kwargs={}),
+            VerifyRehearsal(spy_id=202, spy_name="spy_202", args=(7, 8, 9), kwargs={}),
+        ],
+        calls=[
+            SpyCall(spy_id=101, spy_name="spy_101", args=(0, 0, 0), kwargs={}),
             SpyCall(spy_id=101, spy_name="spy_101", args=(1, 2, 3), kwargs={}),
             SpyCall(spy_id=101, spy_name="spy_101", args=(4, 5, 6), kwargs={}),
             SpyCall(spy_id=202, spy_name="spy_202", args=(7, 8, 9), kwargs={}),

